AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/manage-team/manage-team.component.ts

```diff
@@ -183,6 +183,27 @@ export class ManageTeamComponent implements OnInit, OnDestroy {
         this.selectedUser$.next(null);
         this.userFormMode$.next('add');
     }
+
+    getRoles(roles: JusticeUserRole[]) {
```

Comment:
The component has been kept as clean as possible in the sense that methods to build content to display was kept to a minimum. This would be best to put into a pipe in my opinion

Reply:
moved it to a pipe